stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/channel/ChannelController.kt

```diff
@@ -1121,6 +1136,22 @@ public class ChannelController internal constructor(
                     mute.channel.cid == cid
                 }
             }
+            is ChannelUserBannedEvent,
+            is ChannelUserUnbannedEvent,
+            is NotificationChannelDeletedEvent,
+            is NotificationInvitedEvent,
+            is NotificationRemovedFromChannelEvent,
+            is ConnectedEvent,
+            is ConnectingEvent,
+            is DisconnectedEvent,
+            is ErrorEvent,
+            is GlobalUserBannedEvent,
+            is GlobalUserUnbannedEvent,
+            is HealthEvent,
+            is NotificationMutesUpdatedEvent,
+            is UnknownEvent,
+            is UserDeletedEvent,
+            -> Unit // Ignore these events
```

Comment:
Oh why 😢

Reply:
I didn't want to add an `else` case here, because we handle *most* events, just ignore some specific ones. This way if we add a new one, we're forced to handle it explicitly here.

Reply:
Yeah that's good - it's better to be specific, it's just that we really have a lot of different events and it's hard to keep track of them.